Fix Handling of Undefined Attributes

axkar opened the pull request:
While generating yaml files, the gvpr tool tried to access the values which were not defined in .dot.in files.

This applies to qn_basemac, qn_headmac and qn_tailmac.

troglobit commented on qeneth:
```diff
@@ -217,10 +217,10 @@ genyaml()
-			if (e.qn_tailmac && e.tail == $)
-				printf(" qn_mac %s ", e.qn_tailmac);
+			if (hasAttr(e, "qn_headmac") && e.qn_headmac && e.head == $)
+				printf(" qn_mac: %s ", e.qn_headmac);	
```
Nitpick, extra whitespace introduced at the end of this line.

axkar replied:
thnx, fixed :)

troglobit reviewed (approved):
Looks really good, nice work cleaning up my mistakes! :smile:

Commit:
Fix handling of qn mac attributes for edges

While generating yaml files, the gvpr tool tried to access the
values which were not defined in .dot.in files.

This applies to qn_basemac, qn_headmac and qn_tailmac.

wkz reviewed (approved):
:ship::+1:

(Next time: please open the PR to wkz/qeneth, which is the canonical repo)
